src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -270,8 +235,42 @@ public abstract partial class JsonNode
         public abstract System.Text.Json.JsonNode Clone();
         public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonElement jsonElement) { throw null; }
         public static System.Text.Json.JsonNode GetNode(System.Text.Json.JsonElement jsonElement) { throw null; }
-        public static System.Text.Json.JsonNode Parse(string json) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (bool value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (byte value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.DateTime value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.DateTimeOffset value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (decimal value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (double value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.Guid value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (short value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (int value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (long value) { throw null; }
```

Comment:
What kind of API do you mean exactly?

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/operator-overloads

> ✓ CONSIDER providing methods with friendly names that correspond to each overloaded operator.

Generally, when you have implicit operators, the recommendation is to have a way to accomplish the same thing but by calling APIs with friendly names (as an alternative). Maybe this could be ctor overloads or static methods on JsonNode.

Reply:
I think we do have it - all of the primary types (JsonNumber, JsonString, JsonBoolean) already have such constructors and JsonNumber additionally has getter and setter overloads. Do you want to add them to JsonNode as well? Didn't we want to keep JsonNode's constructor private protected? And JsonNode is an abstract class.